testdata/scripts/basic.txt

```diff
@@ -27,7 +27,7 @@ stdout 'unknown'
 ! stdout $gofullversion
 
 # The binary can't contain the version string either.
-! binsubstr main$exe ${WORK@R} 'globalVar' 'globalFunc' $gofullversion
+! binsubstr main$exe ${WORK@R} 'globalVar' 'globalFunc' $gofullversion 'z0.go'
```

Comment:
Did you test that this fails before your change?

Reply:
You should also test `-tiny`

Reply:
yes, it failed

Reply:
There should also be some kind of test to verify that all line numbers are 1 and filenames are empty strings when `-tiny` is passed

Reply:
Not actually